### DISCUSSION TITLE
My second AD5M, with Klipper mod applied, and DIY enclosure done up nice. + links to my Printables mods so far

### DISCUSSION THREAD

**Discussion by DougJoseph** [xblax/flashforge_ad5m_klipper_mod#122] [Show and tell]
![30 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7558](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/38463e74-557a-4f92-9b14-ea809b9aeec7)

![31 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7557](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/bdb53161-d498-4a07-9a1a-e5e229714e26)

![32 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7556](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/1f7dc4af-7de1-4d57-bbf7-c61c3e2fd08d)

![33 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7559](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/73761dd8-526e-410e-b66a-a7a2e1e06c09)

![34 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7560](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/d6e56e36-cd5a-4231-942c-543cff0eb30b)

![35 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7561](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/641db409-2194-4085-8980-c39592f8698d)

![36 Flashforge Adventurer 5M - my 2nd AD5M done up nice IMG_7564](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/0a2f9eb2-4149-423d-a67f-c05126086be1)

Printables links to my AD5M-related design work so far:
https://www.printables.com/model/785667-flashforge-adventurer-5m-ad5m-simpler-easier-faste
https://www.printables.com/model/813334-flashforge-adventurer-5m-ad5m-printed-camera-mount
https://www.printables.com/model/814546-flashforge-adventurer-5m-ad5m-nozzle-storage-box-t
https://www.printables.com/model/827706-flashforge-adventurer-5-ad5m-led-mod-accessory-buc
https://www.printables.com/model/828632-flashforge-adventurer-5-ad5m-led-light-bars-mod-ac

**Comment by MACCAMENZIES** [comment]
Very nice job
